CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+Your interest in contributing to this project is greatly appreciated. Please know that coding is not the only way to contribute. 
+
+**Here are some ways you can help that doesn't involve coding:**
+
+- Review any active PRs. If the PR needs any changes, please exlpain them in a simple way.
+- You can test the project to see if it has any bugs. If you come across a bug that has not been addressed before, please explain the scenario in a clear and consice manner.
```

Comment:
Also do explain as to how they can list the bugs. Probably add the link to the issues page and ask them to search through the list of issues first to see if it has not been listed before going ahead and adding them.